test/tests/kernels/coupled_time_derivative/coupled_time_derivative_test.i

```diff
@@ -0,0 +1,72 @@
+###########################################################
+# This is a simple test of the CoupledTimeDerivative kernel.
+# The expected solution for the variable v is
+# v(x) = 1/2 * (x^2 + x)
+###########################################################
+
+[Mesh]
+  type = GeneratedMesh
+  nx = 5
+  ny = 5
+  dim = 2
+[]
+
+[Variables]
+  [./u]
+  [../]
+  [./v]
+  [../]
+[]
+
+[Kernels]
+  [./time_u]
+    type = TimeDerivative
+    variable = u
+  [../]
+  [./fn_u]
+    type = UserForcingFunction
+    variable = u
+    function = ffn
+  [../]
+  [./time_v]
+    type = CoupledTimeDerivative
+    variable = v
+    v = u
+  [../]
+  [./diff_v]
+    type = Diffusion
+    variable = v
+  [../]
+[]
+
+[Functions]
+  [./ffn]
+    type = ParsedFunction
```

Comment:
You could pass in `1` as the function name above and not explicitly build a function object here.

Reply:
Didn't know you could do that. Cool